### DISCUSSION TITLE
fission of fusion

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#293] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
alrighty so the nuke failed to explode bc of a safety mechaninism so im remaking it so im wondering fission vs fusion
pick one

**Comment by Noob-ishh** [comment]
fusion

**Comment by PickleIsDaBest** [comment]
fission, it's better lmao.

**Comment by 3kh0** [comment, marked as answer]
stop trying to blow up my house 😠

**Reply by rwg100** [reply]
no
